views/admin/javascripts/exhibits.js

```diff
@@ -468,4 +475,219 @@ Omeka.ExhibitBuilder = {};
             });
         });
     };
+
+    Omeka.ExhibitBuilder.setUpCoverImageChooser = function (coverImageChooserUrl, itemOptionsUrl) {
+        var coverImagePanel = $('#cover-image-panel');
+        var selected_cover_image_id = $('#cover_image_file_id').val();
+
+        coverImagePanel.dialog({
+            autoOpen: false,
+            modal: true,
+            resizable: false,
+            create: function () {
+                $(this).dialog('widget').draggable('option', {
+                    containment: 'window',
+                    scroll: false
+                });
+            },
+            open: function () {
+                function refreshDialog() {
+                    coverImagePanel.dialog('option', {
+                        width: Math.min($(window).width() - 100, 600),
+                        height: Math.min($(window).height() - 100, 500)
+                    });
+                }
+
+                refreshDialog();
+                $('body').css('overflow', 'hidden');
+                $(window).on('resize.ExhibitBuilder', function () {
+                    refreshDialog();
+                });
+            },
+            beforeClose: function () {
+                $('body').css('overflow', 'inherit');
+                $(window).off('resize.ExhibitBuilder');
+                $('#cover-image-item-options').empty();
+            },
+            dialogClass: 'item-dialog'
+        });
+
+        /**
+         * Use AJAX to load the form for an attachment.
+         */
+        function loadItemOptionsForm(data, ajaxUrl) {
```

Comment:
There are two differences.  The first is that the the CSS selectors being targeted in the callback functions are different.  The second is that the earlier function (on line 198) has provisions for loading tinymce which isn't needed in the case of the cover image, although looking at it now it seems like I could probably refactor things to use one, shared function.  Let me give that a shot.